src/pages/en/tutorial/0-introduction/1.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Background Knowledge
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - learned what background programming knowledge you will need to get started building with Astro
+  - chosen and installed any tools that you will use to build your Astro website
+  - set up the necessary online accounts to deploy your site to the web
+</Goals>
+
+This tutorial is designed to be accessible to new web developers, as well as devs who are simply new to Astro. It assumes some basic familiarity with **HTML**, **Markdown**, **CSS**, and a little **JavaScript**. But, you will be able to learn a lot of what you need to know during the tutorial itself!
+
+:::tip
+You can also complete this tutorial using a web browser **instead of installing any local software**! You can access both the command line and a code editor together in a cloud development environment like [StackBlitz](https://stackblitz.com) or [CodeSandbox](https://codesandbox.io).
+
+Visit [astro.new](https://astro.new) and click to open the "Completely Empty" project in the editor of your choice.  "Fork" the project to save it to your own account. Then, skip ahead to setting up [Source Control](#source-control) for your tutorial project.
```

Comment:
Concerns about this flow:
- Source control doesn't work normally in those contexts. In GitPod, you can run git commands like you would on your desktop terminal, but you can't run git commands at all in Stackblitz
- You might want to create your GitHub account _before_ creating a StackBlitz/CSB account, as those often integrate with your GitHub account